README.md

```diff
@@ -60,64 +59,70 @@ aztk spark cluster ssh
 ```bash
-aztk spark cluster get --id <my_cluster_id>
+aztk spark cluster get --id my_cluster
 ```
 
-### Submit a Spark job
+### 3. Submit a Spark job
 
-When your cluster is up, you can submit jobs to run against the cluster:
+When your cluster is ready, you can submit jobs from your local machine to run against the cluster. The output of the spark-submit will be streamed to your local console. Run this command from the cloned AZTK repo:
 ```bash
+// submit a java application
+aztk spark cluster submit \
+    --id my_cluster \
+    --name my_java_job \
+    --class org.apache.spark.examples.SparkPi \
+    --executor-memory 20G \
+    path\to\examples.jar 1000
+    
+// submit a python application
 aztk spark cluster submit \
-    --id <my_cluste_id> \
-    --name <my_job_name> \
-    [options] \
-    <app jar | python file> \
-    [app arguments]
+    --id my_cluster \
+    --name my_python_job \
+    --executor-memory 20G \
+    path\to\pi.py 1000
 ```
-NOTE: The job name (`--name`) must be atleast 3 characters long, can only contain alphanumeric characters including hyphens but excluding underscores, and cannot contain uppercase letters. Each job you submit **must** have a unique name.
+- The `aztk spark cluster submit` command takes the same parameters as the standard [`spark-submit` command](https://spark.apache.org/docs/latest/submitting-applications.html), except instead of specifying `--master`, AZTK requires that you specify your cluster `--id` and a unique job `--name`
+- The job name, `--name`, argument must be atleast 3 characters long
+    - It can only contain alphanumeric characters including hypens but excluding underscores
+    - It cannot contain uppercase letters
+- Each job you submit **must** have a unique name
+- Use the `--no-wait` option for your command to return immediately
 
-The output of spark-submit will be streamed to the console. Use the `--no-wait` option to return immediately. More information regarding monitoring your job can be found in the [spark submit documentation.](./docs/20-spark-submit.md)
+Learn more about the spark submit command [here](./docs/20-spark-submit.md)
 
-To start testing this package, you can start by trying out a Spark job from the [./examples](./examples) folder. The examples are a curated list of samples from Spark-2.2.0.
-
-### Log in and Interact with your Spark Cluster
+### 4. Log in and Interact with your Spark Cluster
 Most users will want to work interactively with their Spark clusters. With the `aztk spark cluster ssh` command, you can SSH into the cluster's master node. This command also helps you port-forward your Spark Web UI and Spark Jobs UI to your local machine:
 ```bash
-aztk spark cluster ssh --id <my_cluster_id>
+aztk spark cluster ssh --id my_cluster --user spark
 ```
 By default, we port forward the Spark Web UI to *localhost:8080*, Spark Jobs UI to *localhost:4040*, and the Spark History Server to *localhost:18080*.
```

Comment:
Worth adding additional ports we are forwarding today? 8787 and 8888 for RStudioServer and Jupyter?